### DISCUSSION TITLE
part-ii/2020-122/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#130] [Question comments]
Part II 2020 Probability and Measure: Paper 3, Section II, 26K 

https://questions.tripos.org/part-ii/2020-122/

**Comment by eha1234** [comment]
I got yes for part (iii)

**Reply by ArthurConmy** [reply]
Yeah, this can be done with the Fourier transform way here https://www.dpmms.cam.ac.uk/study/II/Probability+Measure/2019-2020/lecture-notes-PM.pdf pg. 72. Is there another method?

**Reply by eha1234** [reply]
I used kolmogoriov's 0-1 law, using the random variables we defined in part (ii) :)

**Reply by DannyDuberstein** [reply]
Is the result in those notes about Fourier coefficients vanishing easy to prove?
